src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -367,25 +366,25 @@
     <PrepareForBuildDependsOn>$(PrepareForBuildDependsOn);GetCFamilyFiles</PrepareForBuildDependsOn>
   </PropertyGroup>
 
-  <Target Name="GetCFamilyFiles">
-    <Message Importance="high" Text="Fetching the CFamily files..." />
-
-    <!-- Note: building the helper project explicitly using MSBuild rather than referencing it via a ProjectReference.
-         This makes is easier for us to control when the build is performed, so we can trigger it early enough
-         to include the downloaded files as normal Content files in this project.
-         Also, we are using the "Exec" task to invoke MSBuild.exe rather the "MSBuild" task when building
-         DownloadSonarPlugin.csproj to avoid getting file-locking errors when rebuilding.
+  <Target Name="GetCFamilyFiles" Condition=" $(DesignTimeBuild) != 'true' ">
```

Comment:
Replaces the MSBuild task with the new exe.
If I remember correctly, this code is currently an exact replacement of the MSBuild task i.e. it copies all of the CFamily files to the `lib` directory. It doesn't attempt to consolidate the rule files (I have a separate commit that handles that).